core/server/api/posts.js

```diff
@@ -305,6 +305,99 @@ posts = {
 
             return deletedObj;
         });
+    },
+
+    getTags: function getTags(options) {
```

Comment:
The naming of these methods needs some consideration. Currently we use consistent 'BREAD' names like `api.posts.read()`, not 100% sure at this moment how to extend that pattern to include nested relations. I'm thinking it ought to be something along the lines of: `api.posts.tags.edit()` ?